### DISCUSSION TITLE
Questions about unfixed bugs

### DISCUSSION THREAD

**Discussion by badguy0139** [adtzlr/ttb#31] [Q&A]
Hello, I am very interested in your fortran tensor calculation! Are there any known unfixed bugs in your code? Thank you for your generous answer!

**Comment by adtzlr** [comment]
Hi, 

there are no unresolved bugs that I'm aware of (beside the rudimentary implementation of isotropic tensor functions, see #4; also take care of #8). If you find a bug feel free to report or to contribute! Over the last years there was only one bug discovered, see #18.

**Comment by adtzlr** [comment]
FYI: Recently, I discovered two new bugs, #37 and #40. Beside that, the power function could lead to wrong results, see #36.
